### DISCUSSION TITLE
Response's transcription field contains no carriage returns?

### DISCUSSION THREAD

**Discussion by carlosalvidrez** [deepgram/community#538] [General help] [answered]
Hi! I transcribed a video file using the Deepgram web interface, and it worked great. I can see the text output with multiple paragraphs (carriage returns). I then did the same but using the python library and I get the "transcription" key/result in the response, but I see no sign of carriage returns... 

Is there a way to get well-formed paragraphs using the Python library?

Thanks!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by carlosalvidrez** [reply]
This was the python script I ran: 

`from deepgram import DeepgramClient, PrerecordedOptions

DEEPGRAM_API_KEY = 'asdf'
PATH_TO_FILE = '/Users/XXX/FILE_NAME.mp4'

def deepgram_local_file():
    deepgram = DeepgramClient(DEEPGRAM_API_KEY)

    with open(PATH_TO_FILE, 'rb') as buffer_data:
        payload = {'buffer': buffer_data}
        options = PrerecordedOptions(
            punctuate=True,
            model="nova-2",
            language="es")

        print('Requesting transcript...')
        print('Your file may take up to a couple minutes to process.')
        print('While you wait, did you know that Deepgram accepts over 40 audio file formats? Even MP4s.')
        print('To learn more about customizing your transcripts check out developers.deepgram.com')

        response = deepgram.listen.prerecorded.v('1').transcribe_file(payload, options)
        return response
`

`res = deepgram_local_file()
print(res.to_json(indent=4))`

**Reply by carlosalvidrez** [reply]
Ah, and the request ID:

`"request_id": "c54b8dc7-c802-4368-94b5-34ec3b92a668",`

**Comment by davidvonthenen** [comment, marked as answer]
I think you are looking for this option:
https://developers.deepgram.com/docs/paragraphs

**Reply by carlosalvidrez** [reply]
Yes! Thank you
